src/ucp/api/ucp.h

```diff
@@ -631,6 +633,62 @@ void ucp_worker_progress(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Obtain an event file descriptor for event notification.
+ *
+ * This routine returns a valid file descriptor for polling functions.
+ * The file descriptor   will get signaled when an event occurs.
+ *
+ * @param [in]  worker    Worker of notified events.
+ * @param [out] fd        File descriptor.
+ */
+ucs_status_t ucp_worker_get_efd(ucp_worker_h worker, int *fd_p);
+
+
+/**
+ * @ingroup UCP_WORKER
+ * @brief Turn on event notification for the next event.
+ *
+ * This routine needs to be called for every notification on this worker, so
+ * will typically be called once the processing of the previous event is over.
+ * It must also be called after @ref ucp_worker_event_get_fd for activation.
+ * For example:
```

Comment:
The same here. Why it needs to be called ? Describe the logic.
(once even is consumed user is responsible to re-arm the even ...etc. )